stock_picking_package_preparation_line/models/stock_picking_package_preparation_line.py

```diff
@@ -206,7 +206,8 @@ def action_put_in_pack(self):
             for line in package.line_ids:
                 # ----- If line has 'move_id' this means we don't need to
                 #       recreate picking and move again
-                if line.product_id and not line.move_id:
+                if (line.product_id and line.product_id.type != 'service'
```

Comment:
i always considered that a bug. you should not be able to have a stock of intangible stuff.
Anyway, the decision is up to the authors of the module :-)